docs/posix-secondaries-bitbaking.adoc

```diff
@@ -0,0 +1,114 @@
+and running of which on the target makes it act as `Primary` and `Secondary` ECU of a single device.
+
+Please, refer to link:https://uptane.github.io/uptane-standard/uptane-standard.html[the Uptane standard] in order to grok a meaning of the `Primary` and `Secondary` terms in the given context from a theoretical standpoint.
+Please, check out link:posix-secondaries.doc[this doc] to understand these terms from a practical standpoint and
+to learn how `Primary` and `Secondary` can be emulated locally on an user's host.
+It is highly advisable to follow the doc instructions and play with faked/emulated `Primary` and `Secondary` prior to running steps described further in this doc.
+
+== Bitbaking
+
+It is assumed that a reader is familiar with Yocto and bitbaking in general as well as bitbaking of Aktualizr images in particular,
+details of which are out of scope of this doc and can be found in the following guides:
+
+* link:https://github.com/advancedtelematic/meta-updater/blob/master/README.adoc[meta-updater README].
+* https://docs.ota.here.com/quickstarts/qemuvirtualbox.html[How to build 'core-image-minimal' image for QEMU]
+
+=== Primary
+To bitbake an image for `Primary` run the following:
+....
+bitbake primary-image
+....
+
+
+.Primary configuration variables
+[cols="1,1,10"]
+|===
+|Name |Default |Description
+
+|`PRIMARY_IP`
+|`"10.0.3.1"`
+|An IP address to assign to one of the Primary's NIC for communication with Secondaries
+
+|`PRIMARY_PORT`
+|`"9040"`
+| A TCP port that Primary aktualizr listen on for connections from Secondaries
+
+|`PRIMARY_WAIT_TIMEOUT`
+|`"120"`
+|Time (seconds) to wait for connections from Secondaries. Only the secondaries that connected to Primary will be registered at the server and are part of the device Primary represents.
+
+|`PRIMARY_SECONDARIES`
+|`"10.0.3.2:9050"`
+| A space separated list of TCP/IP addresses of the Secondaries to be included into the list of ECUs served by the given Primary
+|===
+
+
+=== Secondary
+To bitbake an image for `Secondary` run the following
+....
+bitbake secondary-image
+....
+
+.Secondary configuration variables
+[cols="1,1,10"]
+|===
+|Name |Default |Description
+
+|`SECONDARY_IP`
+|`"10.0.3.2"`
+|An IP address to assign to A Secondary NIC for communication with Primary
+
+|`SECONDARY_PORT`
+|`"9050"`
+|A TCP port that Secondary listen on for connections from Primary
+|===
+
+
+== Running
+
+It is assumed that a reader is familiar with details on running of bitbaked images targeted for QEMU, such information can be found in the following docs:
+
+* link:https://github.com/advancedtelematic/meta-updater/blob/master/README.adoc[meta-updater README].
+* https://docs.ota.here.com/quickstarts/qemuvirtualbox.html[How to build 'core-image-minimal' image for QEMU]
+
+=== Primary
+
+To launch QEMU VM acting as Primary run the following from your build directory
+....
+sudo ../meta-updater/scripts/run-qemu-ota --no-gui --secondary-network primary-image
```

Comment:
Is sudo required because of the network setup?

Reply:
No, it's not required, I still use it due to some historical reason. Will remove 'sudo' from the doc.